from classes.exception.BaseAppException import BaseAppException

class ValidationException(BaseAppException):
    pass
